.github/workflows/ci.yml

```diff
@@ -81,9 +81,15 @@ jobs:
             $f || exit 1
           done
 
+      - name: Run GUI tests
```

Comment:
I wonder if this should actually be a separate job, running in parallel to `tests` and `build_tests`

Reply:
we probably could also use `upload-artifact` above to upload the `cratesfyi` binary above to achieve this.

Reply:
or perhaps rather part of the `build_tests`, so the inital build setup is only done once, we could test this

Reply:
So which one should I do? :)

Reply:
Ok I'll put it into its own testsuite then.